### PR TITLE
Sign up

### PR DESCRIPTION
**Pull request by kalpeshDocFox**

### CODE REVIEW

File: app/views/users/new.html.erb [hunk 1]
```diff
@@ -1,2 +1,16 @@
-<h1>Users#new</h1>
-<p>Find me in app/views/users/new.html.erb</p>
+<% provide(:title, 'Sign up') %>
+<h1>Sign up</h1>
+<div class="row">
+  <div class="col-md-6 col-md-offset-3">
+    <%= form_with(model: @user, local: true) do |f| %> <%= render 'shared/error_messages' %>
```
Comment (ldmaselela) [inline app/views/users/new.html.erb]: Please place the `render` in a new line

File: test/integration/users_signup_test.rb [hunk 2]
```diff
@@ -0,0 +1,25 @@
+require 'test_helper'
+class UsersSignupTest < ActionDispatch::IntegrationTest
+  test "validate that signups that dont meet form criteria fail" do
+    get signup_path
+    assert_no_difference 'User.count' do
+      post users_path, params: { user: { name:  "",
```
Comment (ldmaselela) [inline test/integration/users_signup_test.rb]: Please consider indenting this object using just one tab

File: test/integration/users_signup_test.rb [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+                                         password: "foo",
+                                         password_confirmation: "bar" } }
+    end
+    assert_template 'users/new'
+  end
+
+  test "test to confirm with valid signup information" do
+    get signup_path
+    assert_difference 'User.count', 1 do
+      post users_path, params: { user: { name: "Example User",
```
Comment (ldmaselela) [inline test/integration/users_signup_test.rb]: Tiddo on indenting

**Review by ldmaselela — APPROVED** [review]

File: test/integration/users_signup_test.rb [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+    assert_no_difference 'User.count' do
+      post users_path, params: { user: { name:  "",
+                                         email: "user@invalid",
+                                         password: "foo",
+                                         password_confirmation: "bar" } }
+    end
+    assert_template 'users/new'
+  end
+
+  test "test to confirm with valid signup information" do
```
Comment (c13e) [inline test/integration/users_signup_test.rb:16]: see similar feedback on Oratile's PR https://github.com/oratilemlambo/sample_app/pull/5/files#r757303625

**Review by c13e — APPROVED** [review]
Noice!

**Pull request merged**